wavefront/config/wavefront.yaml

```diff
@@ -10,5 +10,5 @@ spec:
ZnVsbHkgcXVhbGlmaWVkIElzdGlvIG1ldHJpYyBpbnN0YW5jZSBuYW1lLgoKDwoHBAADAgIBBBIEVwRUFAoOCgcEAAMCAgEFEgNXBAoKDgoHBAADAgIBARIDVwsYCg4KBwQAAwICAQMSA1cbHAohCgYEAAMCAgISA1oEEhoSIFRoZSBtZXRyaWMgdHlwZS4KCg8KBwQAAwICAgQSBFoEVx0KDgoHBAADAgICBhIDWgQICg4KBwQAAwICAgESA1oJDQoOCgcEAAMCAgIDEgNaEBEKaQoGBAADAgQAEgRdBGgFGlkgRGVzY3JpYmVzIG1ldHJpYyB0eXBlcyBhcyBpbiBbV2F2ZWZyb250XShodHRwczovL2RvY3Mud2F2ZWZyb250LmNvbS9tZXRyaWNfdHlwZXMuaHRtbCkuCgoOCgcEAAMCBAABEgNdCQ0KNQoIBAADAgQAAgASA18GEhokIFJlc2VydmVkIGZvciB1bmtub3duIG1ldHJpYyB0eXBlcy4KChAKCQQAAwIEAAIAARIDXwYNChAKCQQAAwIEAAIAAhIDXxARCjIKCAQAAwIEAAIBEgNhBhAaISBSZXByZXNlbnRzIGEgZ2F1Z2UgbWV0cmljIHR5cGUuCgoQCgkEAAMCBAACAQESA2EGCwoQCgkEAAMCBAACAQISA2EODwo0CggEAAMCBAACAhIDYwYSGiMgUmVwcmVzZW50cyBhIGNvdW50ZXIgbWV0cmljIHR5cGUuCgoQCgkEAAMCBAACAgESA2MGDQoQCgkEAAMCBAACAgISA2MQEQo6CggEAAMCBAACAxIDZQYYGikgUmVwcmVzZW50cyBhIGRlbHRhIGNvdW50ZXIgbWV0cmljIHR5cGUuCgoQCgkEAAMCBAACAwESA2UGEwoQCgkEAAMCBAACAwISA2UWFwo2CggEAAMCBAACBBIDZwYUGiUgUmVwcmVzZW50cyBhIGhpc3RvZ3JhbSBtZXRyaWMgdHlwZS4KChAKCQQAAwIEAAIEARIDZwYPChAKCQQAAwIEAAIEAhIDZxITClcKBgQAAwICAxIDawQWGkggRm9yIG1ldHJpY3Mgd2l0aCB0eXBlIEhJU1RPR1JBTSwgdGhpcyBkZXNjcmliZXMgdGhlIHNhbXBsZSBkZWZpbml0aW9uLgoKDwoHBAADAgIDBBIEawRoBQoOCgcEAAMCAgMGEgNrBAoKDgoHBAADAgIDARIDawsRCg4KBwQAAwICAwMSA2sUFQp2CgYEAAMCAwASBW4EhgEFGmUgRGVzY3JpYmVzIGEgc2FtcGxlIGFzIGluIHRoZSBbcmNyb3dsZXkvZ28tbWV0cmljc10oaHR0cHM6Ly9naXRodWIuY29tL3Jjcm93bGV5L2dvLW1ldHJpY3MpIGxpYnJhcnkuCgoOCgcEAAMCAwABEgNuDBIKQgoIBAADAgMACAASBHAGdgcaMCBPbmUgb2YgdGhlIHNhbXBsZSBkZWZpbml0aW9uIG11c3QgYmUgc3VwcGxpZWQuCgoQCgkEAAMCAwAIAAESA3AMFgpCCggEAAMCAwACABIDcggfGjEgRGVmaW5pdGlvbiBvZiBhbiBleHBvbmVudGlhbGx5IGRlY2F5aW5nIHNhbXBsZS4KChAKCQQAAwIDAAIABhIDcggQChAKCQQAAwIDAAIAARIDchEaChAKCQQAAwIDAAIAAxIDch0eCjIKCAQAAwIDAAIBEgN1CBwaISBEZWZpbml0aW9uIG9mIGEgdW5pZm9ybSBzYW1wbGUuCgoQCgkEAAMCAwACAQYSA3UIDwoQCgkEAAMCAwACAQESA3UQFwoQCgkEAAMCAwACAQMSA3UaGwo/CggEAAMCAwADABIEeQZ/BxotIERlc2NyaWJlcyBhbiBleHBvbmVudGlhbGx5IGRlY2F5aW5nIHNhbXBsZS4KChAKCQQAAwIDAAMAARIDeQ4WCigKCgQAAwIDAAMAAgASA3sIIRoVIFRoZSByZXNlcnZvaXIgc2l6ZS4KChMKCwQAAwIDAAMAAgAEEgR7CHkYChIKCwQAAwIDAAMAAgAFEgN7CA0KEgoLBAADAgMAAwACAAESA3sOHAoSCgsEAAMCAwADAAIAAxIDex8gCh8KCgQAAwIDAAMAAgESA34IGRoMIFRoZSBhbHBoYS4KChMKCwQAAwIDAAMAAgEEEgR+CHshChIKCwQAAwIDAAMAAgEFEgN+CA4KEgoLBAADAgMAAwACAQESA34PFAoSCgsEAAMCAwADAAIBAxIDfhcYCjEKCAQAAwIDAAMBEgaCAQaFAQcaHSBEZXNjcmliZXMgYSB1bmlmb3JtIHNhbXBsZS4KChEKCQQAAwIDAAMBARIEggEOFQopCgoEAAMCAwADAQIAEgSEAQghGhUgVGhlIHJlc2Vydm9pciBzaXplLgoKFQoLBAADAgMAAwECAAQSBoQBCIIBFwoTCgsEAAMCAwADAQIABRIEhAEIDQoTCgsEAAMCAwADAQIAARIEhAEOHAoTCgsEAAMCAwADAQIAAxIEhAEfIAo2CgQEAAMDEgaKAQKNAQMaJiBEZXNjcmliZXMgdGhlIGxvZ2dpbmcgY29uZmlndXJhdGlvbi4KCg0KBQQAAwMBEgSKAQoOClQKBgQAAwMCABIEjAEEFRpEIFRoZSBsb2cgbGV2ZWwgKG9uZSBvZiBlcnJvciwgd2FybiwgaW5mbywgZGVidWcsIG9yIG5vbmUpLiBFeDogaW5mbwoKEQoHBAADAwIABBIGjAEEigEQCg8KBwQAAwMCAAUSBIwBBAoKDwoHBAADAwIAARIEjAELEAoPCgcEAAMDAgADEgSMARMUCiYKBAQAAgYSBJABAhAaGCBUaGUgbG9nIGNvbmZpZ3VyYXRpb24uCgoPCgUEAAIGBBIGkAECjQEDCg0KBQQAAgYGEgSQAQIGCg0KBQQAAgYBEgSQAQcLCg0KBQQAAgYDEgSQAQ4PYgZwcm90bzM=
```

Comment:
Can you revert all changes in the `wavefront/config` directory? Are any of those required?

Reply:
Reverted the changes and made config files same as master. However observed that these files are getting changed when build is run.